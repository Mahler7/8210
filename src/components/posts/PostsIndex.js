import React from 'react';

const PostsIndex = () => (
  <h1>Posts Index</h1>
);

export default PostsIndex;