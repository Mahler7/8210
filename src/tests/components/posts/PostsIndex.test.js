import React from 'react';
import { shallow } from 'enzyme';
import PostsIndex from '../../../components/posts/PostsIndex';

test('should render PostsIndex page', () => {
  const wrapper = shallow(<PostsIndex/>);
  expect(wrapper).toMatchSnapshot();
});