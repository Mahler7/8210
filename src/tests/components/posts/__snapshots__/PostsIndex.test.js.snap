// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PostsIndex page 1`] = `
<h1>
  Posts Index
</h1>
`;
